samples/drivers/flash_shell/src/main.c

```diff
@@ -299,7 +299,7 @@ static int do_write_unaligned(const struct shell *shell, off_t offset, uint8_t *
 	if (!single_page_write) {
 		size_t middle_data_len = aligned_size;
 		off_t middle_page_start = start_page;
-		off_t data_offset = (off_t)buf;
+		off_t data_offset = (uintptr_t)buf;
```

Comment:
Hi @carlocaione,

The mcuboot maintainer is not responsive. https://github.com/mcu-tools/mcuboot/pull/1413
Wonder if I can just upstream this patch?